### PR TITLE
Add visualization to active channels

### PR DESCRIPTION
**Pull request by liannivin**

### CODE REVIEW

File: public/main.html [hunk 1]
```diff
@@ -16,7 +16,12 @@
     <section id="channel-list" class="channelList">
       <div class="channels">
         <h3>User Channels</h3>
-        <ul id="user_channels"></ul>
+        <ul id="user_channels">
+          <li class="canal">Canal 1</li>
+          <li class="canal">Canal 2</li>
+          <li class="canal">Canal 3</li>
+          <li class="canal">Canal 4</li>
+        </ul>
```
Comment (cristianbgp) [inline public/main.html]: I think this is just for testing, we should delete them in the future

File: public/assets/js/main.js [hunk 2]
```diff
@@ -182,3 +181,14 @@ pushingIrcChannels = data => {
     $channelChild.innerHTML = data;
   }
 };
+
+// listar canales activos
+const $listaMensajes = document.getElementById("js-messages-view");
+document.querySelectorAll(".canal").forEach(function(p) {
```
Comment (luciagirasoles) [inline public/assets/js/main.js]: I think this might be "channel" instead of "canal"

File: public/assets/js/main.js [hunk 3]
```diff
@@ -182,3 +181,14 @@ pushingIrcChannels = data => {
     $channelChild.innerHTML = data;
   }
 };
+
+// listar canales activos
```
Comment (luciagirasoles) [inline public/assets/js/main.js]: Maybe ```//List Active Channels```

File: public/assets/js/main.js (hunk 2 above)
Comment (liannivin) [reply public/assets/js/main.js]: Fixed. Thanks for your observation.

File: public/main.html (hunk 1 above)
Comment (cdsandoval) [reply public/main.html]: > I think this is just for testing, we should delete them in the future

I agree with this. We must delete the list of li. Using this kind of code for testing is a normal thing but try to delete it before ask for pull request.

File: public/assets/js/main.js [hunk 4]
```diff
@@ -182,3 +181,14 @@ pushingIrcChannels = data => {
...
+const $listActiveChannels = document.getElementById("js-messages-view");
+document.querySelectorAll(".channel").forEach(function(p) {
```
Comment (cdsandoval) [inline public/assets/js/main.js]: Please, try to use a variable with sense. Would recommend to change the p insteand you can use "element" or anything else with sense.

**Pull request merged**